apps/client/src/pages/dashboard/components/ui/indicators/Timer.tsx

```diff
@@ -1,41 +1,41 @@
-import { memo } from 'react';
-import { useServerTimer } from '@shared/hooks/useServerTimer';
+import { memo, useEffect, useState } from 'react';
 
 interface TimerProps {
   onComplete?: () => void;
+  reminderHour?: number;
 }
 
-const Timer = ({ onComplete }: TimerProps) => {
-  const { formattedTime, isLoading, error, timerState } =
-    useServerTimer(onComplete);
-
-  const getTimerText = () => {
-    if (isLoading) {
-      return '로딩 중...';
-    }
-    if (error) {
-      return '시간 로드 실패';
-    }
-    if (timerState.isExpired) {
-      return '타이머 완료';
-    }
-    return formattedTime;
-  };
-
-  const getTimerColor = () => {
-    if (isLoading) {
-      return 'text-gray400';
-    }
-    if (error) {
-      return 'text-error400';
-    }
-    if (timerState.isExpired) {
-      return 'text-gray400';
-    }
-    return 'text-gray400';
-  };
-
-  return <span className={`sub3-m ${getTimerColor()}`}>{getTimerText()}</span>;
+const Timer = ({ onComplete: _onComplete, reminderHour = 9 }: TimerProps) => {
+  const [timeLeft, setTimeLeft] = useState<string>('00:00:00');
```

Comment:
`useState`에 초기 값을 넣어주면 `타입 추론`으로 인해서 따로 제니릭을 넣어주지 않아도 괜찮습니다! 물론 가독성을 위해서 넣는 것도 방법이지만, `타입 추론`을 적극적으로 사용하는 것도 좋은 방법이 될 것 같아요 👍 

```suggestion
const [timeLeft, setTimeLeft] = useState('00:00:00');
```

Reply:
타입 추론을 활용하도록 수정해두었습니다! 감사합니다:)